src/translators/sql_translator.py

```diff
@@ -1551,10 +1551,10 @@ def query_entity_types(self, fiware_service=None, fiware_servicepath=None):
             for et in all_types:
                 stmt = "select distinct(entity_type) from {}".format(et)
                 if fiware_servicepath == '/':
-                    stmt = stmt + " WHERE {} ~* '/.*'" \
+                    stmt = stmt + " WHERE {} = '/'" \
```

Comment:
not sure about this changes

Reply:
most probably if your revert this, tests should pass

Reply:
I have removed ```.*``` as this results in extracting all the entities. I have also updated the if block as the statement can be used in both the cases.

Reply:
@chicco785 ```.*``` should not be the behavior expected from QL if we want it to behave in the same manner as that of ```orion```. Please consider my comment  https://github.com/orchestracities/ngsi-timeseries-api/pull/564#issuecomment-920820646 and let me know if I need to updated test cases in translator to match this behavior.

Reply:
I have reverted my changes. We can consider points which I have mentioned in https://github.com/orchestracities/ngsi-timeseries-api/pull/564#issuecomment-920820646 in separate issue so its functionality matches ```orion``` broker. Or you have any suggestion please correct me.